pkgs/desktops/plasma-5/powerdevil.nix

```diff
@@ -18,4 +18,10 @@ mkDerivation {
   cmakeFlags = [
     "-DHAVE_DDCUTIL=On"
   ];
+  patches = [
+    # This fixes an issue where 'DDCA_Feature_List*' cannot be converted to
+    # 'DDCA_Feature_List'.
+    # This can probably be dropped on next update, maybe.
```

Comment:
Would be nice to say what the upstreaming story is. Is the issue known upstream?

Reply:
Reported upstream as https://bugs.kde.org/show_bug.cgi?id=423605. I’m not sure where one would send a patch, since https://phabricator.kde.org/source/powerdevil/ is marked “Inactive”.

Reply:
https://invent.kde.org/plasma/powerdevil

Reply:
Great, please link the bug here, then we're ready go to!

Reply:
Sorry it took me a while to get back to this. There was a commit 5 hours ago to the aforementioned invent.kde.org project, so I've opted to `fetchpatch` that: https://invent.kde.org/plasma/powerdevil/-/commit/fcb26be2fb279e6ad3b7b814d26a5921d16201eb